### PR TITLE
fix: run all button

### PR DESCRIPTION
**Pull request by sjain-21**
added run all button

### CODE REVIEW

File: zt_frontend/src/App.vue [hunk 1]
```diff
@@ -115,6 +123,7 @@
         :notebook="notebook"
         :completions="completions"
         @runCode="runCode"
+        @runAllCode="runAllCode"
```
Comment (Carson-Shaar) [inline zt_frontend/src/App.vue:126]: Was it necessary to add another method? Was just making a call to runCode with empty originId not working? You would have to adjust the runCode method to accept this but not sure if another method is necessary

Comment (Carson-Shaar) [reply zt_frontend/src/App.vue:126]: Also this is for emitted events so should not be needed here. Only on the button

File: zt_backend/runner/execute_code.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def execute_request(request: request.Request, state: UserState):
             downstream_cells = [cell.id for cell in request.cells if cell.cellType in ['code', 'sql']]
         for code_cell_id in downstream_cells:
             code_cell = dependency_graph.cells[code_cell_id]
-            if code_cell_id!=request.originId and code_cell.nonReactive:
+            if request.originId and code_cell_id != request.originId and code_cell.nonReactive:
```
Comment (Carson-Shaar) [inline zt_backend/runner/execute_code.py:85]: Can you explain this one to me? I believe there was already handling here for an empty originId?

File: zt_frontend/src/App.vue (hunk 1 above)
Comment (sjain-21) [reply zt_frontend/src/App.vue]: Removed runAllCode

File: zt_backend/runner/execute_code.py (hunk 2 above)
Comment (sjain-21) [reply zt_backend/runner/execute_code.py]: Not require, I just mentioned it explicitly. Removed it.

**Pull request closed without merging**